pkg/dashboard/ui/src/app/shared/services/nuclio-functions-data.service.js

```diff
@@ -40,7 +41,7 @@
 
             var config = {
                 method: 'post',
-                url: NuclioClientService.buildUrlWithPath('functions'),
+                url: NuclioClientService.buildUrlWithPath('functions', importProcess ? '?import=true' : ''),
```

Comment:
Since we don't use the default function parameters, we can use `lodash.defaultTo(value, defaultValue)` method like this:
```suggestion
                url: NuclioClientService.buildUrlWithPath('functions', lodash.defaultTo(importProcess, false) ? '?import=true' : ''),
```

_But_, I don't advise you to pass the param like that.
Instead of putting it directly on the URL, you can pass it in the config.
If you follow this `config` object here that goes later into `NuclioClientService.makeRequest` it eventually passed into a call to `$http` which performs the request itself.
As you can see in the [`$http` API docs](https://docs.angularjs.org/api/ng/service/$http#$http-arguments) this config can have a `params` property whose value is a key-value object of the parameters you want to pass in the `GET` request.
Then what we need here is:
```js
{
    import: true
}
```
`$http` service uses the `$httpParamSerializer` service to convert this object into the final string, as you can see in its [API docs](https://docs.angularjs.org/api/ng/service/$httpParamSerializer). If it gets `undefined`, `null` or the empty string `''` it would just not add any parameter to the request (no `?` at all). Also, if any property has `undefined` or `null` value, its key will not be added as a parameter. For example, the object `{ foo: 'bar', baz: null }` will result in `?foo=bar`.
So, my final suggestion is:
```suggestion
                url: NuclioClientService.buildUrlWithPath('functions'),
                params: {
                    import: lodash.defaultTo(importProcess, false)
                },
```